config/token.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"github.com/dgrijalva/jwt-go"
+	"github.com/kiali/kiali/log"
+	"time"
```

Comment:
minor, but @jmazzitelli will be happy if we order the imports in
standard imports
<space>
third party imports
<space>
kiali imports

Reply:
Changed, we should put this in the future style code guide ^^ Thanks @lucasponce